buster/chatbot.py

```diff
@@ -10,11 +10,8 @@
 from openai.embeddings_utils import cosine_similarity, get_embedding
 
 from buster.docparser import read_documents
-from buster.formatter import Formatter, HTMLFormatter, MarkdownFormatter, SlackFormatter
-from buster.formatter.base import Response, Source
-
-FORMATTERS = {"text": Formatter, "slack": SlackFormatter, "html": HTMLFormatter, "markdown": MarkdownFormatter}
-
+from buster.formatter.base import Response, ResponseFormatter, Source
```

Comment:
Importing directly from the module, rather that the implementation detail `base` would make more sense, I think. 
```suggestion
from buster.formatter import Response, ResponseFormatter, Source
```

Reply:
fixed in the next commit